File: woodroid/src/main/java/me/gilo/woodroid/data/callbacks/CategoriesCallback.java
package me.gilo.woodroid.data.callbacks;


import com.google.gson.annotations.SerializedName;
import me.gilo.woodroid.models.Category;


import java.util.ArrayList;


public class CategoriesCallback {
    @SerializedName("product_categories")
    ArrayList<Category> categories;

    public ArrayList<Category> getCategories() {
        return categories;
    }

    public void setCategories(ArrayList<Category> categories) {
        this.categories = categories;
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/data/callbacks/CategoriesCallback.kt
package me.gilo.woodroid.data.callbacks


import com.google.gson.annotations.SerializedName
import me.gilo.woodroid.models.Category


import java.util.ArrayList


class CategoriesCallback {
    @SerializedName("product_categories")
    lateinit var categories: ArrayList<Category>
}
